R/writeManifest.R

```diff
@@ -35,7 +35,7 @@ writeManifest <- function(appDir = getwd(),
     appDir = appDir,
     appFiles = appFiles,
     appPrimaryDoc = appPrimaryDoc,
-    quarto = quarto,
+    usesQuarto = quarto,
```

Comment:
Does `writeManifest` need the same deprecation warning?

Reply:
Good catch; I'll move the deprecation into `inferAppMetadata()`